.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
+.vscode/*
```

Comment:
Since this repo is just filled with markdown files we shouldn't need a `.gitignore`  file.
Let's remove this